definition/VERSION.yaml

```diff
@@ -1 +1 @@
-TAG: 1.2.6
+TAG: 1.2.7
```

Comment:
we shouldnt bump version for this - just for definition updates...can you update the github PR notice pls 😊

Reply:
oh sure 👍